pkg/dumper/postgres/dumper.go

```diff
@@ -72,10 +81,40 @@ func (d *pgDumper) DumpTable(tableName string, rowChan <-chan database.Row) erro
 // PreDumpTables Disable triggers on all tables to avoid foreign key constraints
 func (d *pgDumper) PreDumpTables(tables []string) error {
 	// We can't use `SET session_replication_role = replica` because multiple connections and stuff
-	for _, tbl := range tables {
-		query := fmt.Sprintf("ALTER TABLE %s DISABLE TRIGGER ALL", strconv.Quote(tbl))
-		if _, err := d.conn.Exec(query); err != nil {
-			return errors.Wrapf(err, "Failed to disable triggers for %s", tbl)
+	// For RDS databases, the superuser does not have the required permission to call
+	// DISABLE TRIGGER ALL, so manually remove and re-add all Foreign Keys
+	if d.isRDS {
```

Comment:
Please use early return here to minimise code nesting. Non-RDS logic is simpler so it may be nested, something like:

```go
if !d.isRDS {
  log.Debug("Disabling triggers")
  ...
  return nil
}

log.Debug("Removing foreign keys")
...
```